karma.conf.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
```

Comment:
Yes you can definitely do all these things manually or with bunch of other tools. It is just simpler to outsource all this work to something else. Like Karma. I can definitely explore other ways more (I did it before an it was not as simple as I would have thought), but I doubt it will be more expressive and less difficult in the result. Will almost certainly include some hacking with tools to mix and match them together.

I understand your concerns with locking yourself into some ecosystem, though find it reasonable in the case.